pkg/kube/utils.go

```diff
@@ -160,3 +160,14 @@ func IsNodeReady(node *v1.Node) bool {
 	}
 	return false
 }
+
+// IsAccessModesOfPVCContainReadOnly return true if AccessModes of PVC contain `ReadOnlyMany`
+func IsAccessModesOfPVCContainReadOnly(pvc *v1.PersistentVolumeClaim) bool {
```

Comment:
The suggestion that I provided will be able to do exactly what the current function is doing without any issues, but it can also be used from other places as well. For example if someone wants to test a PVC contains a specific accessMode or not.

Reply:
It breaks [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle, what is no good IMO. Another thing, that by my experience, too flexible functions could lead to bug, which will be very difficult to find. In this particular case, after implementing of your suggestion
```suggestion
func PVCHasAccessMode(pvc *v1.PersistentVolumeClaim, mode AccessMode) bool {
```
will be possible to accidentally pass incorrect access mode to this function, what will break a code actually.
Summarize, I understand your motivation, but I think leaving this function as is makes code more robust and simple. Thus, if you don't mind, I'd prefer don't apply your suggestion here.

Reply:
No problem at all @k0taperk0t, we can keep it as it is.
But just for arguments sake :laughing: , if we think that caller can pass incorrect value as argument to a function in that case any written function would be problematic right?

Reply:
Also, what do you think about the name `PVCContainsReadOnlyAccessMode` instead of  `IsAccessModesOfPVCContainReadOnly`. Totally optional again.

Reply:
@viveksinghggits , you're right, if caller will pass incorrect value to this function it will not problem of function. But unfortunately it will became our problem when this code will doesn't work, since we are maintainers of this code. 
A lot of best practices from most popular OOP languages, such as C++ and C#, aren't using in Go, but at the same time, some things teats as anti-patterns in C++ & C#, is Go recommended way. For example, functions for boolean logic in many languages starts from Is/Has/Should, but it's anti-pattern for Go. So..ok, I'll rename `IsAccessModesOfPVCContainReadOnly` to `PVCContainsReadOnlyAccessMode`.

Reply:
fixed